### PR TITLE
[EDU-3115] feat: add secure application

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page with Secure an application content.

Note: Some links aren't working yet.

[EDU-3115]: https://aziontech.atlassian.net/browse/EDU-3115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/secure-application.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+meta_tags: edge, security, application
+namespace: docs_secure_application
+menu_namespace: secureMenu
+permalink: /documentation/products/secure/secure-application/
+---
+
+import Button from '~/components/Button.astro'
+import Badge from '~/components/Badge.astro'
+
+[Edge Firewall](/en/documentation/products/edge-firewall/) allows you to secure your [edge applications](/en/documentation/products/guides/build/build-an-application/) and [domains](/en/documentation/products/guides/configure-a-domain/) against several types of threats and attacks.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:13]: Achei muitos links para um texto bem curto...
talvez tirar o de EF deixa melhor?

File: src/content/docs/en/pages/secure-journey/secure-application.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Azion provides a range of tools to configure your edge firewall:
+
+- [Real-Time Manager (RTM)](https://manager.azion.com/): provides a user-friendly interface to configure your edge firewall and rule sets.
+- [Azion API](https://api.azion.com/#00268e07-463f-4de1-9b45-ceeab187bf19): allows you to create, configure, and automate edge firewalls and its related modules and features.
+- [Azion SDK](): allows you to improve the development of your Go application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:19]:
```suggestion
- [Azion SDK](/en/documentation/devtools/sdk/go/): allows you to improve the development of your Go application.
```

File: src/content/docs/en/pages/secure-journey/secure-application.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [Terraform](/en/documentation/products/secure/automate/terraform/): provides integration for your infrastructure with Azion using Terraform. Configurations available for main settings and edge functions instances.
+- [Edge Runtime](): leverages APIs for networking and data handling.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:21]:
```suggestion
- [Edge Runtime](/en/documentation/products/edge-application/edge-functions/runtime/overview/): leverages APIs for networking and data handling.
```

File: src/content/docs/en/pages/secure-journey/secure-application.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Step 2: Choose a function to instantiate
+
+The second step is to instantiate an edge function.
+
+Requires:
+
+<Badge variant="accent">[Edge Functions for Edge Firewall](/en/documentation/products/edge-firewall/edge-functions/firewall/)</Badge>
+
+Instantiating an existing edge function created for security purposes, such as the **Bot Protection** or **Bot Management** integrations, reinforces your security policies and adds another layer of protection for your applications.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:48]:
```suggestion
Instantiating an existing edge function created for security purposes, such as the **Bot Protection** or **Scheduled Blocking** integrations, reinforces your security policies and adds another layer of protection for your applications.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/secure-application.mdx (hunk 1 above)
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:13]:
```suggestion
**Edge Firewall** allows you to secure your [edge applications](/en/documentation/products/guides/build/build-an-application/) and [domains](/en/documentation/products/guides/configure-a-domain/) against several types of threats and attacks.
```

File: src/content/docs/en/pages/secure-journey/secure-application.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+## Step 1: Configure main settings
+
+To create an edge firewall, the first step is to provide:
+
+- A unique and easy-to-remember name.
+- The domains you want to associate with the edge firewall.
+- The modules you want to use.
+    - This is an optional step, but to fully secure your applications, it's recommend to enable all available modules.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/secure-application.mdx:32]:
```suggestion
    - This is an optional step, but to fully secure your applications, it's recommended to enable all available modules.
```

**Review by gabriel-azion — APPROVED** [review]
- mind the links 
- a "typo"

**Pull request merged**